src/sentry/migrations/0772_backfill_grouphash_metadata_grouping_config.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 5.1.1 on 2024-10-01 00:47
+
+from django.apps.registry import Apps
+from django.db import migrations
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+
+from sentry.new_migrations.migrations import CheckedMigration
+
+DEFAULT_GROUPING_CONFIG = "newstyle:2023-01-11"
+
+
+def fill_in_missing_grouping_config(apps: Apps, schema_editor: BaseDatabaseSchemaEditor) -> None:
+    GroupHashMetadata = apps.get_model("sentry", "GroupHashMetadata")
+
+    for gh_metadata in GroupHashMetadata.objects.filter(latest_grouping_config=None):
+        gh_metadata.update(latest_grouping_config=DEFAULT_GROUPING_CONFIG)
```

Comment:
Unfortunately since we're in a migration, we don't have access to the `.update` method, since that's a custom method we define on our models.

The equivalent of this is
```
gh_metadata.latest_grouping_config = DEFAULT_GROUPING_CONFIG
gh_metadata.save(update_fields=["latest_grouping_config"])
```

Reply:
Got it - I'd missed the fact that `<object>.update` was a convenience method from us rather than from Django, and was being lazy about writing it out in two steps. I'll go change it.